File: mifosng-android/src/main/java/com/mifos/api/model/ScorecardPayload.java
/*
 * This project is licensed under the open source MPL V2.
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */
package com.mifos.api.model;

import com.mifos.objects.survey.ScorecardValues;

import java.util.Date;
import java.util.List;

/**
 * Created by Nasim Banu on 28,January,2016.
 */
public class ScorecardPayload {
    private int userId;
    private int clientId;
    private Date createdOn;
    private List<ScorecardValues> scorecardValues;


    public int getUserId() {
        return userId;
    }

    public void setUserId(int userId) {
        this.userId = userId;
    }

    public int getClientId() {
        return clientId;
    }

    public void setClientId(int clientId) {
        this.clientId = clientId;
    }

    public Date getCreatedOn() {
        return createdOn;
    }

    public void setCreatedOn(Date createdOn) {
        this.createdOn = createdOn;
    }

    public List<ScorecardValues> getScorecardValues() {
        return scorecardValues;
    }

    public void setScorecardValues(List<ScorecardValues> scorecardValues) {
        this.scorecardValues = scorecardValues;
    }
}

File: core/network/src/main/java/com/mifos/core/network/model/ScorecardPayload.kt
/*
 * Copyright 2024 Mifos Initiative
 *
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at https://mozilla.org/MPL/2.0/.
 *
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */
package com.mifos.core.network.model

import com.mifos.core.objects.survey.ScorecardValues
import java.util.Date

/**
 * Created by Nasim Banu on 28,January,2016.
 */
data class ScorecardPayload(
    var userId: Int = 0,
    var clientId: Int = 0,
    var createdOn: Date? = null,
    var scorecardValues: List<ScorecardValues>? = null,
)
